sources/Renderer/QtRenderer/QtSFML.cpp

```diff
@@ -7,35 +7,38 @@
 #include <QIcon>
 #include <QMenuBar>
 #include <iostream>
+#include <QDir>
+#include <QFileDialog>
+#include "Models/Logger.hpp"
 #include "SNES.hpp"
-#include "../../Exceptions/DebuggableError.hpp"
 #include "QtSFML.hpp"
 
 #ifdef Q_WS_X11
-	#include <Qt/qx11info_x11.h>
-	#include <X11/Xlib.h>
+#include <Qt/qx11info_x11.h>
```

Comment:
no indent ?

Reply:
Preprocessor statements are actually never indented by convention. No linter support this, so I decided to remove those for consistency.